pmlc/rt/vulkan/vulkan_runtime.cc

```diff
@@ -903,6 +946,16 @@ LogicalResult VulkanRuntime::createSchedule() {
   RETURN_ON_VULKAN_ERROR(vkBeginCommandBuffer(commandBuffer, &beginInfo),
                          "vkBeginCommandBuffer");
 
+  /*
+  void vkCmdWriteTimestamp(
+    VkCommandBuffer                             commandBuffer,
+    VkPipelineStageFlagBits                     pipelineStage,
+    VkQueryPool                                 queryPool,
+    uint32_t                                    query);
+  */
+  vkCmdWriteTimestamp(commandBuffer, VK_PIPELINE_STAGE_TOP_OF_PIPE_BIT,
```

Comment:
I think it is good. In compute pipeline, there are not many stages, basically top->draw indirect->compute shader->bottom. I think using VK_PIPELINE_STAGE_TOP_OF_PIPE_BIT or compute or bottom does not make much difference.

Reply:
I think you are right.  Doesn't seem to matter.  I went with VK_PIPELINE_STAGE_COMPUTE_SHADER_BIT for consistency with the kernel actions.